draft-ietf-httpbis-digest-headers.md

```diff
@@ -1059,7 +1065,13 @@ in conjuction with the encrypted content-coding {{?RFC8188}}.
 
 ## Algorithm Agility
 
-...
+Algorithm agility is achieved establishing the IANA Digest Algorithm Values registry,
+and adding it the "Status" field making it possible to deprecate and obsolete insecure algorithms.
+
+The now obsoleted [RFC3230] allowed passing parameters to digest-algorithms:
+if those parameters are used to verify the checksum, there will be enough space for bad or insecure
+implementations. If a digest-algorithm want to support parameters, it can encode them into the
+representation-data-digest.
```

Comment:
These paragraphs are confusing under the section title. I think it would be clearer to 

1) Take what's in the first paragraph and combine it with what's in section 12.2 and 12.3. I suggest we do that in a different PR

2) Rename the section to something like "Paramaterising digest-algorithms"

3) Take the second paragraph and explain a little more why the old means was "bad or insecure". For example, what could have gone wrong? That makes the problem statement clearer so that the solution of embedding parameters make more sense to anyone considering what to do.

Reply:
Sure, I'll move 
```
Algorithm agility is achieved establishing the IANA Digest Algorithm Values registry,
and adding it the "Status" field making it possible to deprecate and obsolete insecure algorithms.
```
to another PR!

Reply:
I propose to move all this section in {{obsolete-parameters}} as removing parameters we do not have security considerations.

Reply:
About combining 12.2 and 12.3, I re-read rfc 8126 and iiuc those sections should remain unchanged ("keep iana considerations for iana")... Let's discuss it in https://github.com/httpwg/http-extensions/pull/1284